stackslib/src/net/api/setloglevel.rs

```diff
@@ -0,0 +1,164 @@
+// Copyright (C) 2013-2020 Blockstack PBC, a public benefit corporation
```

Comment:
This seems very unrelated to this PR?

Reply:
Also I think we should update the ChangeLog to include it if we do plan to merge this as part of this PR.